src/communication/network_interface_sr.c

```diff
@@ -7281,27 +7281,40 @@ slogtb_dump_trantable (THREAD_ENTRY * thread_p, unsigned int rid, char *request,
 void
 smmon_get_server_info (THREAD_ENTRY * thread_p, unsigned int rid, char *request, int reqlen)
 {
-  char *buffer = NULL, *ptr;
+  char *buffer_a = NULL, *ptr, *buffer;
   int size = 0;
   OR_ALIGNED_BUF (OR_INT_SIZE + OR_INT_SIZE) a_reply;
   char *reply = OR_ALIGNED_BUF_START (a_reply);
   int error = NO_ERROR;
   MMON_SERVER_INFO server_info;
+  int string_len;
 
   mmon_aggregate_server_info (server_info);
 
-  size += or_packed_string_length (server_info.name, NULL);
+  string_len = or_packed_string_length (server_info.name, NULL);
+  if (string_len % MAX_ALIGNMENT)
```

Comment:
We need a comment describing why it's needed.

Reply:
Done.

Reply:
The comment seems missed here.